xarray/tests/test_formatting.py

```diff
@@ -405,10 +405,19 @@ def test_short_numpy_repr():
         np.random.randn(20, 20),
         np.random.randn(5, 10, 15),
         np.random.randn(5, 10, 15, 3),
+        np.random.randn(100, 5, 1),
     ]
     # number of lines:
-    # for default numpy repr: 167, 140, 254, 248
-    # for short_numpy_repr: 1, 7, 24, 19
+    # for default numpy repr: 167, 140, 254, 248, 599
+    # for short_numpy_repr: 1, 7, 24, 19, 30
```

Comment:
why is the last case shortened to 30 lines? When I try it, I get (using the line limit of `40`): `41`. The tests seem to pass, though, so I might be missing something.

Reply:
ah, my bad, that test is about `short_numpy_repr`, not `short_data_repr`.

Reply:
Yes, do you maybe have different numpy defaults?

Reply:
`np.get_printoptions()` says
```
{'edgeitems': 3,
 'threshold': 1000,
 'floatmode': 'maxprec',
 'precision': 8,
 'suppress': False,
 'linewidth': 75,
 'nanstr': 'nan',
 'infstr': 'inf',
 'sign': '-',
 'formatter': None,
 'legacy': False}
```
I can't remember setting anything, so these should be the generic defaults.

Reply:
You're right! I'm not sure how I made this mistake; I get 25 too.